src/components/Select.jsx

```diff
@@ -0,0 +1,213 @@
+          doubleWord = true;
+        } else {
+          doubleWord = false;
+        }
+        workSquares = loadCell(posX, posY, workSquares, doubleWord, workWordNo);
+        if (alignment === 'row') {
+          posX++;
+        } else {
+          posY++;
+        }
+        // randomly adjust if you build the next word on the current word's last or 2nd last letter
+        //      row/1st letter of new word is last letter of previous/ 4 or 5 letter word/not 1st word
+        if (alignment === 'row' && x === 1 && wordLengths[i - 1] > 2 && i > 0 && wordLengths[i] !== 2) {
+          if (randomNumber === 1) {
+            posY--;
+            workSquares = switchCell(workSquares);
+          }
+        } else if (
+          alignment === 'column' &&
+          x === 1 &&
+          wordLengths[i - 1] > 2 &&
+          i > 0 &&
+          wordLengths[i] !== 2
+        ) {
+          if (randomNumber === 1) {
+            posX--;
+            workSquares = switchCell(workSquares);
+          }
+        }
+      }
+      // switch direction of next word
+      if (alignment === 'row') {
+        alignment = 'column';
+      } else {
+        alignment = 'row';
+      }
+      // if room still exists for another word, start processing word lengths at the beginning
+      if (alignment === 'row' && posX + wordLengths[0] < 9 && i + 1 === wordLengths.length) {
+        i = -1;
+      } else if (alignment === 'column' && posY + wordLengths[0] < 9 && i + 1 === wordLengths.length) {
+        i = -1;
+      }
+      // increment word counter
+      workWordNo++;
+    }
+    workWordNo--;
+    let doubleWord = false;
+    workSquares = loadCell(posX, posY, workSquares, doubleWord, workWordNo);
+    setWordNo(workWordNo);
+    return workSquares;
+  };
+
+  const saveClicked = () => {
+    if (numberSelected === '') {
+      setErrorMessage('Number of letters  not selected');
+      return;
+    }
+    if (wordLengths.length === 0) {
+      setErrorMessage('Lengths of words  not selected');
+      return;
+    }
+    setErrorMessage('');
+    setSelectNumber(true);
+    let workSquares = loadBoard();
+    // code to load a test board
+    if (useTestBoard) {
+      workSquares = JSON.parse(JSON.stringify(testBoard));
+    }
+    setSquares(workSquares);
+  };
+
+  return (
+    <>
+      <div className="instructions">
+        Enter a number betwen 1 & 20 which
+        <br />
+        limits the number of consonents you 
+        <br />
+        can play. Select the word sizes you
+        <br />
+        want the board to be composed of.
+        <br />
+      </div>
+      <br />
+      <div className="inputdev">
+        <input
+          required
+          name="value"
+          className="selected"
+          type="text"
+          value={numberSelected}
+          maxLength="2"
+          onChange={editInput}
+        />
+        <span className="instructionsNum">Enter number here</span>
+      </div>
+      <div>
+        <form>
```

Comment:
Is the `form` tag required here? `form` is a pretty old HTML element, I don't think its generally recommended anymore